--- udev.orig	2019-06-16 14:56:16.711099348 +0000
+++ udev	2019-06-16 14:14:30.952735417 +0000
@@ -21,7 +21,7 @@
 	log_level=info
 fi
 
-SYSTEMD_LOG_LEVEL=$log_level /lib/systemd/systemd-udevd --daemon --resolve-names=never
+SYSTEMD_LOG_LEVEL=$log_level /lib/systemd/systemd-udevd --debug --resolve-names=never &
 
 udevadm trigger --type=subsystems --action=add
 udevadm trigger --type=devices --action=add
